template/base/_.prettierignore

```diff
@@ -14,4 +14,5 @@ node_modules*/
 /.gantry/**/*.yml
 gantry*.yaml
 gantry*.yml
+cdk.out*/
```

Comment:
1. Can you avoid smushing this together with the Gantry ignores? It doesn't not seem related and is probably more at home alongside the first block with `node_modules` and such.
2. Is the asterisk necessary?